packages/ckeditor5-table/src/ui/inserttableview.js

```diff
@@ -158,7 +158,8 @@ export default class InsertTableView extends View {
 			keystrokeHandler: this.keystrokes,
 			focusTracker: this.focusTracker,
 			gridItems: this.items,
-			numberOfColumns: 10
+			numberOfColumns: 10,
+			uiLanguageDirection: this.locale.uiLanguageDirection
```

Comment:
I did some digging and I see that `locale` is optional in all the packages that are already rewritten to TS, so we can safely assume it will happen also for this one - let's add the check you've provided elsewhere: `this.locale && this.locale.uiLanguageDirection`.